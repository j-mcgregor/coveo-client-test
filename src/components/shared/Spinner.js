/* eslint-disable import/prefer-default-export */
import React from 'react';
import S from '../../static/styles';

export default () => (<S.Spinner />);
