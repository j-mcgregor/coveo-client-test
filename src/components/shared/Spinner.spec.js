import React from 'react';
import { shallow } from 'enzyme';
import Spinner from './Spinner';

describe('Spinner', () => {
  it('should render the Spinner', () => {
    const wrapper = shallow(<Spinner />);

    expect(wrapper).toMatchSnapshot();
  });
});
