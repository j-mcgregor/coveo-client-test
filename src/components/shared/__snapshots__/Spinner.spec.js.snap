// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner should render the Spinner 1`] = `ShallowWrapper {}`;
